ngDraggable.js

```diff
@@ -375,14 +375,21 @@ angular.module("ngDraggable", [])
                     }
                 };
 
-                var hitTest = function(x, y) {
+               var hitTest = function(x, y) {
                     var bounds = element[0].getBoundingClientRect();// ngDraggable.getPrivOffset(element);
                     x -= $document[0].body.scrollLeft + $document[0].documentElement.scrollLeft;
                     y -= $document[0].body.scrollTop + $document[0].documentElement.scrollTop;
+                
+                    var draggingElmt = document.getElementsByClassName("dragging")[0];
```

Comment:
There's a bug here. When a user clicks on a draggable element without dragging it an exception is thrown in the console.